.travis.yml

```diff
@@ -9,20 +9,26 @@ cache:
   bundler: true
   directories:
     - node_modules
+matrix:
+  include:
+    - osx_image: xcode9.4
+      env: BUILD_SDK=iphonesimulator11.4 DESTINATION="platform=iOS Simulator,name=iPhone 8" MAIN_STAGE="true"
+    - osx_image: xcode8.3
+      env: BUILD_SDK=iphonesimulator10.3 DESTINATION="platform=iOS Simulator,name=iPhone 7" SNAPSHOT_TESTS="false" MAIN_STAGE="false"
 before_install:
   - nvm install 8.9.4
-  - npm install -g greenkeeper-lockfile@1
+  - '[ "$MAIN_STAGE" = true ] && npm install -g greenkeeper-lockfile@1 || true'
```

Comment:
I think the naming here is good, but just wondering if we should have a comment at the top of the file or something to explain _why_ we only run tests on >= iOS 11